### ISSUE TITLE
Printrboard Bleutooth Serial

### ISSUE THREAD

**Issue by ghost** [MarlinFirmware/Marlin#7110]
Hello,

Firmware only compile when the following headers are added to serial.h after enabling bluetooth serial:
```
#include "MarlinSerial.h"
#include"Arduino.h"
#include"macros.h"

```

Firmware after this modification and compilation works ok.

Edit1: To add USB doesnt work, though serial does with this. Even if bluetooth_serial is disabled, unless I remove these above.

**Comment by Bob-the-Kuhn** [comment]
Try these changes in serial.h.  For me PRINTRBOARD compiled with Bluetooth , compiled & ran without bluetooth and RAMPS compiled & ran just fine.  I don't have the ability to test the Bluetooth option.
```
#include "MarlinSerial.h"  // added
//#include"Arduino.h"  // not needed
//#include"macros.h"  // not needed

#ifdef USBCON
//  #include "HardwareSerial.h"  // not needed
  #if ENABLED(BLUETOOTH)
    #define MYSERIAL bluetoothSerial
  #else
    #define MYSERIAL Serial
  #endif // BLUETOOTH
#else
//  #include "MarlinSerial.h"   // not needed
  #define MYSERIAL customizedSerial
#endif
```

**Comment by thinkyhead** [comment]
Now patched in 1.1.x.
